### DISCUSSION TITLE
Issue when renaming configmaps

### DISCUSSION THREAD

**Discussion by rmuller-vu** [mia-platform/community#563] [Q&A]
### What product are you having troubles with?

Console

### What Console version are you using?

1.102.8

### Description

We are running into an issue when we want to rename a config map to have a different casing.

We have a configmap file that is named `casing-test.json`. We realized our name is supposed to be `Casing-test.json` with a capital `C`. If we delete the lowercase one and create the uppercase one it will show an error when we save saying duplicate naming.

![image](https://github.com/user-attachments/assets/fa087516-e1c7-425f-8888-4dc77579999d)

The work around we use is to save after deleting the lowercase file then adding the new capitalized file

**Comment by marcofilippi** [comment]
Hi @rmuller-vu 

Thanks for bringing this to our attention. It seems that indeed different casing within ConfigMap files is not handled correctly.

We’ve tracked the issue and will investigate it promptly.

**Reply by KarissaBrummell** [reply]
Hello! I wanted to follow up on this because it still seems to be an issue: 
<img width="1103" height="801" alt="image" src="https://github.com/user-attachments/assets/edeaeb26-5aa6-4cdd-a154-beb9746db7a5" />

**Comment by KarissaBrummell** [comment]
Hello! We are now on Mia-Platform Suite Version v14.3.0, Console Version 1.119.11 and this problem is resolved! We can close out this discussion.
